monitoring/notification/dashboard.py

```diff
@@ -0,0 +1,567 @@
+            '))',
+            '/',
+            'sum by (job) (rate (',
+            ' ' + Metrics.CACHE_LAG.count(),
+            ')), "job", "$1", "job", "' + LABEL_EXPR['notification-processors'] +'")',
+        ]),
+        legendFormat='{{job}}',
+    )],
+    thresholdType='absolute',
+    thresholds=[
+        Threshold('green', 0, 0.),
+    ])
+
+dashboard = (
+    Dashboard(
+        title='Bucket Notifications',
+        editable=True,
+        refresh='30s',
+        tags=['backbeat', 'notification'],
+        timezone='',
+        inputs=[
+            DataSourceInput(
+                name='DS_PROMETHEUS',
+                label='Prometheus',
+                pluginId='prometheus',
+                pluginName='Prometheus',
+            ),
+            ConstantInput(
+                name='namespace',
+                label='namespace',
+                description='Namespace associated with the Zenko instance',
+                value='zenko',
+            ),
+            ConstantInput(
+                name='job_notification_producer',
+                label='notification producer',
+                description='Name of the notification producer job to filter metrics',
+                value='artesca-data-backbeat-notification-producer-headless',
+            ),
+            ConstantInput(
+                name='job_notification_processors',
+                label='notification processors',
+                description='Name of the notification processor jobs to filter metrics',
+                value='artesca-data-backbeat-notification-processor',
+            ),
+            ConstantInput(
+                name='job_zookeeper',
+                label='zookeeper quorom',
+                description='Name of the zookeeper jobs to filter metrics',
+                value='artesca-data-base-quorum-headless',
+            ),
+            ConstantInput(
+                name='job_kafka',
+                label='kafka brokers',
+                description='Name of the kafka jobs to filter metrics',
+                value='artesca-data-base-queue',
+            ),
+            ConstantInput(
+                name='notification_topics',
+                label='bucket notification topics',
+                description='Name of the bucket notification kafka topics',
+                value='.*backbeat-notification.*',
+            ),
+        ],
+        templating=Templating([
+            Template(
+                dataSource='${DS_PROMETHEUS}',
+                label='Notification processor',
+                name='notification_processor',
+                query='label_values(' + Metrics.PROCESSOR_EVENTS.raw('job=~"${job_notification_processors}.*"') + ', job)',
+                regex='/^(?<value>(' + DEFAULT_JOB_PREFIX + ')?(backbeat-notification-processor-)?(?<text>.*?)(-headless)?)$/',
+                allValue='${job_notification_processors}.*',
+                includeAll=True,
+                multi=True,
+            )
```

Comment:
may be nice to have a "target" variable here?
(and would just need to add a default value for target label in each metric : `target=~"$target"`)